api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -447,16 +447,14 @@ message HttpConnectionManager {
   config.core.v3.HttpProtocolOptions common_http_protocol_options = 35
       [(udpa.annotations.security).configure_for_untrusted_downstream = true];
 
-  // If :ref:`common_http_protocol_options.max_connection_duration
-  // <envoy_v3_api_field_config.core.v3.HttpProtocolOptions.max_connection_duration>` is
-  // set and this is set to true, the ``max_connection_duration`` setting
-  // will not cause Envoy to ever close HTTP1 connections (other settings are unaffected). Instead,
-  // Envoy will start adding the Connection:close response header to all responses and let the
-  // client choose when to close the connection.
+  // If set to true, :ref:`common_http_protocol_options.max_connection_duration
+  // <envoy_v3_api_field_config.core.v3.HttpProtocolOptions.max_connection_duration>` will not
+  // trigger Envoy to close HTTP1 connections. Instead, Envoy will only add the Connection:close
+  // response header to all responses until the client chooses to close the connection.
```

Comment:
Can you update this as it's now compliant with spec and will wait for one more request before sending the `connection:close` header.